### PR TITLE
Add meeting minutes for 2018-02-22

### LINKED ISSUE #245: Node.js Foundation Community Committee Meeting 2018-02-22
## Time

**UTC Thu 22-Feb-2018 17:00 (05:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 22-Feb-2018 09:00 (09:00 AM) |
| US / Mountain | Thu 22-Feb-2018 10:00 (10:00 AM) |
| US / Central  | Thu 22-Feb-2018 11:00 (11:00 AM) |
| US / Eastern  | Thu 22-Feb-2018 12:00 (12:00 PM) |
| London        | Thu 22-Feb-2018 17:00 (05:00 PM) |
| Amsterdam     | Thu 22-Feb-2018 18:00 (06:00 PM) |
| Moscow        | Thu 22-Feb-2018 20:00 (08:00 PM) |
| Chennai       | Thu 22-Feb-2018 22:30 (10:30 PM) |
| Hangzhou      | Fri 23-Feb-2018 01:00 (01:00 AM) |
| Tokyo         | Fri 23-Feb-2018 02:00 (02:00 AM) |
| Sydney        | Fri 23-Feb-2018 04:00 (04:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Community%20Committee+Meeting+2018-02-22&iso=20180222T17
* or http://www.wolframalpha.com/input/?i=05PM+UTC%2C+Feb+22%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1ZeWyJLVu4-e-urEPGNW6MnjiGxeXFtbT7i6RC7CGiks/edit>

## Agenda

Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/TSC

* reaching out to irccloud.com [#292](https://github.com/nodejs/TSC/issues/292)

### nodejs/community-committee

* Meeting for reviewing membership guidelines [#234](https://github.com/nodejs/community-committee/issues/234)
* Current Strategic Initiatives Review [#222](https://github.com/nodejs/community-committee/issues/222)
* Issuing badges for contributors [#212](https://github.com/nodejs/community-committee/issues/212)
* Explain inclusivity better [#117](https://github.com/nodejs/community-committee/issues/117)

## Invited

* CommComm Members: @nodejs/community-committee

### Observers
* @therebelrobot (Oz Haven - observer)
* @ParidelPooya (Pooya Paridel - observer)
* @rykerrumsey (Ryker Rumsey - observer)
* @baez (Behzad Karim - observer)
* @BinarySo1o (Therese Stirling - observer)
* @amiller-gh (Adam Miller - observer)
* @yosuke-furukawa (Yosuke Furukawa - observer)
* @Maurice-Hayward (Maurice Hayward - observer)
* @mikehostetler (Mike Hostetler - observer)
* @sarahkconway (Sarah Conway - observer)
* @Tiriel (Benjamin Zaslavsky - observer)
* @DiegoZoracKy (Diego ZoracKy - observer)
* @ZibbyKeaton (Zibby Keaton - observer)
* @tobyfarley (Toby Farley - observer)
* @Bamieh (Ahmad Bamieh - observer)
* @pchrysa (Chrysa - observer)
* @foadlab (observer)
* @codeekage (Abraham Agiri - observer)
* @Voskan (Voskan Voskanyan - observer)

## Notes

The agenda comes from issues labelled with `cc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: Will be added a few minutes before meeting starts
* For those who just want to watch: 
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

### PR DESCRIPTION
**Pull request by bnb**
Add meeting minutes for 2018-02-22.

Closes #245

### CODE REVIEW

File: meetings/2018-02-22.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+## Links
+
+* **Recording**: https://youtu.be/uZKB27_Xu4A
+* **GitHub Issue**: https://github.com/nodejs/community-committee/issues/245
+* **Minutes Google Doc**: https://docs.google.com/document/d/1ZeWyJLVu4-e-urEPGNW6MnjiGxeXFtbT7i6RC7CGiks/edit?usp=sharing
+
+## Present
+- Ben Michel  @obensource
+- Tierney Cyren  @bnb Maintainer
```
Comment (oe) [inline meetings/2018-02-22.md]: maintainer?

File: meetings/2018-02-22.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+- Joe Sepi  @joesepi
+- Zibby Keaton @ZibbyKeaton
+
+@Bamieh (Ahmad Bamieh - observer)
+@amiller-gh (Adam Miller - observer)
+@Maurice-Hayward (Maurice Hayward - observer)
+@tobyfarley (Toby Farley - observer)
+@ZibbyKeaton (Zibby Keaton - observer)
+
+* CommComm Members: @nodejs/community-committee
```
Comment (oe) [inline meetings/2018-02-22.md]: some degree of duplication here

File: meetings/2018-02-22.md (hunk 1 above)
Comment (bnb) [reply meetings/2018-02-22.md]: No clue ¯\_(ツ)_/¯

File: meetings/2018-02-22.md (hunk 2 above)
Comment (bnb) [reply meetings/2018-02-22.md]: Should I drop members/invited observers? 🤔

Comment (oe) [reply meetings/2018-02-22.md]: @bnb I think so. It's in the original issue anyways.

File: meetings/2018-02-22.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+* **Minutes Google Doc**: https://docs.google.com/document/d/1ZeWyJLVu4-e-urEPGNW6MnjiGxeXFtbT7i6RC7CGiks/edit?usp=sharing
+
+## Present
+- Ben Michel  @obensource
+- Tierney Cyren  @bnb Maintainer
+- Olivia  @oe
+- Manil  @chowdhurian
+- Jem Bezooyen  @JemBijoux
+- Joe Sepi  @joesepi
+- Zibby Keaton @ZibbyKeaton
```
Comment (mhdawson) [inline meetings/2018-02-22.md]: I seem to be missing 
- Michael Dawson @nodejs/evangelism 

I was chairing so probably missed adding myself to the list

**Comment by Tiriel** [comment]
Just noticed the PR mentions `2018-01-25` but the minutes are for the  `2018-02-22`.
No big deal, but I just thought it should be clarified!

**Comment by bnb** [comment]
@Tiriel thanks for noticing that, updated.

**Comment by bnb** [comment]
Fixed the comments, @mhdawson @oe could you take a look again?

File: meetings/2018-02-22.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+- Jem Bezooyen  @JemBijoux
+- Joe Sepi  @joesepi
+- Michael Dawson @mhdawson 
+- @ZibbyKeaton (Zibby Keaton - observer)
+- @Bamieh (Ahmad Bamieh - observer)
+- @amiller-gh (Adam Miller - observer)
+- @Maurice-Hayward (Maurice Hayward - observer)
+- @tobyfarley (Toby Farley - observer)
+- @ZibbyKeaton (Zibby Keaton - observer)
+
```
Comment (WaleedAshraf) [inline meetings/2018-02-22.md]: @waleedashraf  (Waleed Ashraf - observer)
I added myself in docs https://docs.google.com/document/d/1ZeWyJLVu4-e-urEPGNW6MnjiGxeXFtbT7i6RC7CGiks
at meeting start time.

**Comment by bnb** [comment]
Going to merge this 👍

**Pull request merged**